connect_ext_ppr/models/utils.py

```diff
@@ -4,35 +4,35 @@
 
 def add_deployments(installation, listings, config, logger):
     with get_db_ctx_manager(config) as db:
-        print(db.bind)
         deployments = []
+        seen = set()
         for li in listings:
             product_id = li['product']['id']
             for hub in li['contract']['marketplace']['hubs']:
+                comb = (product_id, installation['owner']['id'], hub['hub']['id'])
                 q = db.query(Deployment).filter_by(
                     product_id=product_id,
                     account_id=installation['owner']['id'],
                     hub_id=hub['hub']['id'],
                 )
                 if db.query(q.exists()).scalar():
                     continue
-                dep = Deployment(
-                    product_id=product_id,
-                    hub_id=hub['hub']['id'],
-                    vendor_id=li['vendor']['id'],
-                    account_id=installation['owner']['id'],
-                )
-                logger.info(
-                    f"Generating Deployment: product={product_id},"
-                    f" account={installation['owner']['id']}, hub={hub['hub']['id']}.",
-                )
-                deployments.append(dep)
+                if comb not in seen:
+                    dep = Deployment(
+                        product_id=product_id,
+                        hub_id=hub['hub']['id'],
+                        vendor_id=li['vendor']['id'],
+                        account_id=installation['owner']['id'],
+                    )
+                    logger.info(
+                        f"Generating Deployment: product={product_id},"
```

Comment:
[COD-C] Let's use single quotes?

Reply:
One convention that we have about this on the connect is to use double quotes for strings displayed for the user, I think is not an issue to maintain this for extensions.

Reply:
Ok